package.json

```diff
@@ -29,7 +29,7 @@
 		"@nextcloud/router": "^1.0.2",
 		"@nextcloud/vue": "^1.5.0",
 		"escape-html": "^1.0.3",
-		"highlight.js": "^9.18.0",
+		"highlight.js": "^9.16.2",
```

Comment:
If the problem is in highlight.js > 9.16, shouldn't the version be specified as `< 9.17` instead? `^` also includes minor release updates, so next time that dependabot runs it will try to update highlight.js again to 9.18.x. Or am I missing something? :-)

Reply:
Just that I'd like dependabot to bug me about the version :wink: Also we have the package lock comitted, so the version there will be used when installing with npm ci.

Reply:
> Just that I'd like dependabot to bug me about the version :wink:

Hehe, fair enough ;-)